### DISCUSSION TITLE
Pet Tracker

### DISCUSSION THREAD

**Discussion by MrsKayC** [runelite/runelite#18090] [Ideas]
In addition to boss kcs and skill levels it would be nice to see the pets people have. They're a great achievement and you can only have 1 follow at a time. So the only way to let show your accomplishment is with the command in chat !pets or for people to visit your poh.

**Comment by Nightfirecat** [comment]
The hiscores plugin can only display data which is found on the osrs hiscores. If you want pets to be displayed, Jagex would need to add it there.
